src/output-json-dns.c

```diff
@@ -56,6 +56,69 @@
  * TX id handling doesn't expect it */
 #define QUERY 0
 
+#define LOG_QUERIES   (1<<0)
+#define LOG_ANSWERS   (1<<1)
+
+#define LOG_A         (1<<2)
+#define LOG_NS        (1<<3)
+#define LOG_CNAME     (1<<4)
+#define LOG_SOA       (1<<5)
+#define LOG_PTR       (1<<6)
+#define LOG_MX        (1<<7)
+#define LOG_TXT       (1<<8)
+#define LOG_AAAA      (1<<9)
+#define LOG_SRV       (1<<10)
+#define LOG_NAPTR     (1<<11)
+#define LOG_DS        (1<<12)
+#define LOG_RRSIG     (1<<13)
+#define LOG_NSEC      (1<<14)
+#define LOG_NSEC3     (1<<15)
+#define LOG_TKEY      (1<<16)
+#define LOG_TSIG      (1<<17)
+
+#define LOG_ALL_RRTYPES (~(LOG_QUERIES|LOG_ANSWERS))
+
+typedef enum {
```

Comment:
Isnt this a duplication of the data provided here :https://github.com/inliniac/suricata/blob/6286e7055521875a150c7c7c5ff0a715ede27a36/src/app-layer-dns-common.h#L36

This leads us to having two places to update the RRTYPEs